Assets/MultiBuild/Editor/Settings.cs

```diff
@@ -22,13 +22,16 @@ public enum Target {
         PS4 = 12,
         XboxOne = 13,
         SamsungTV = 14,
-        Nintendo3DS = 15,
-        WiiU = 16,
-        tvOS = 17,
-#if UNITY_5_6
-        Switch = 18
+        WiiU = 15,
```

Comment:
Because the values in this enum are serialised in the saved settings, they should not be altered; you've changed the values of these 3 while you were editing, please revert them to their original values before I can merge this.